var/spack/repos/builtin/packages/chombo/package.py

```diff
@@ -0,0 +1,140 @@
+    depends_on('hdf5+mpi', when='+mpi+hdf5')
+
+    def edit(self, spec, prefix):
+
+        # Set fortran name mangling in Make.defs
+        defs_file = FileFilter('./lib/mk/Make.defs')
+        defs_file.filter('^\s*#\s*cppcallsfort\s*=\s*',
+                         'cppcallsfort = -DCH_FORT_UNDERSCORE')
+
+        # Set remaining variables in Make.defs.local
+        # Make.defs.local.template.patch ensures lines for USE_TIMER,
+        # USE_LAPACK and lapackincflags are present
+        copyfile('./lib/mk/Make.defs.local.template',
+                 './lib/mk/Make.defs.local')
+
+        defs_file = FileFilter('./lib/mk/Make.defs.local')
+
+        # Unconditional settings
+        defs_file.filter('^\s*#\s*DEBUG\s*=\s*', 'DEBUG = FALSE')
+        defs_file.filter('^\s*#\s*OPT\s*=\s*', 'OPT = TRUE')
+        defs_file.filter('^\s*#\s*PIC\s*=\s*', 'PIC = TRUE')
+        # timer code frequently fails compiles. So disable it.
+        defs_file.filter('^\s*#\s*USE_TIMER\s*=\s*', 'USE_TIMER = FALSE')
+
+        # LAPACK setup
+        lapack_blas = spec['lapack'].libs + spec['blas'].libs
+        defs_file.filter('^\s*#\s*USE_LAPACK\s*=\s*', 'USE_LAPACK = TRUE')
+        defs_file.filter(
+            '^\s*#\s*lapackincflags\s*=\s*',
+            'lapackincflags = -I%s' % spec['lapack'].prefix.include)
+        defs_file.filter(
+            '^\s*#\s*syslibflags\s*=\s*',
+            'syslibflags = %s' % lapack_blas.ld_flags)
+
+        # Compilers and Compiler flags
+        defs_file.filter('^\s*#\s*CXX\s*=\s*', 'CXX = %s' % spack_cxx)
+        defs_file.filter('^\s*#\s*FC\s*=\s*', 'FC = %s' % spack_fc)
+        if '+mpi' in spec:
+            defs_file.filter(
+                '^\s*#\s*MPICXX\s*=\s*',
+                'MPICXX = %s' % self.spec['mpi'].mpicxx)
+
+        # Conditionally determined settings
+        defs_file.filter(
+            '^\s*#\s*MPI\s*=\s*',
+            'MPI = %s' % ('TRUE' if '+mpi' in spec else 'FALSE'))
+        defs_file.filter(
+            '^\s*#\s*DIM\s*=\s*',
+            'DIM = %s' % spec.variants['dims'].value)
+
+        # HDF5 settings
+        if '+hdf5' in spec:
+            defs_file.filter('^\s*#\s*USE_HDF5\s*=\s*', 'USE_HDF5 = TRUE')
+            defs_file.filter(
+                '^\s*#\s*HDFINCFLAGS\s*=.*',
+                'HDFINCFLAGS = -I%s' % spec['hdf5'].prefix.include)
+            defs_file.filter(
+                '^\s*#\s*HDFLIBFLAGS\s*=.*',
+                'HDFLIBFLAGS = %s' % spec['hdf5'].libs.ld_flags)
+            if '+mpi' in spec:
+                defs_file.filter(
+                    '^\s*#\s*HDFMPIINCFLAGS\s*=.*',
+                    'HDFMPIINCFLAGS = -I%s' % spec['hdf5'].prefix.include)
+                defs_file.filter(
+                    '^\s*#\s*HDFMPILIBFLAGS\s*=.*',
+                    'HDFMPILIBFLAGS = %s' % spec['hdf5'].libs.ld_flags)
+
+    def build(self, spec, prefix):
+        with working_dir('lib'):
+            make('all')
```

Comment:
Yes, good catch! You'll have to use `gmake()` instead of `make()`. These are defined [here](https://github.com/LLNL/spack/blob/develop/lib/spack/spack/build_environment.py#L394).